### PR TITLE
minor improvements to the distance UI

### LINKED ISSUE #38: Improve display of location distance

### PR DESCRIPTION
**Pull request by resir014**
Move distance to the bottom to prepare for future redesign. Fixes #38

image

### CODE REVIEW

File: src/components/VaxLocation.tsx [hunk 1]
```diff
@@ -118,11 +110,24 @@ export default function VaxLocation({ loading, location, isUserLocationExist })
             </WrapItem>
           ))}
         </Wrap>
-        <Tooltip hasArrow label={new Date(lastUpdated).toString()}>
-          <Text align="right" as="i" color={colorMode === 'dark' ? 'gray.300' : 'gray.600'}>
-            Diperbarui {formatDistanceToNow(Date.parse(lastUpdated), { locale: idLocale, addSuffix: true })}
-          </Text>
-        </Tooltip>
+        <Grid templateColumns="1fr 3fr" gridTemplateAreas={`"distance timestamp"`} gap={2}>
```
Comment (pveyes) [inline src/components/VaxLocation.tsx]: TIL

**Review by pveyes — APPROVED** [review]

**Review by sonnylazuardi — APPROVED** [review]
LGTM 👍

**Comment by tibudiyanto** [comment]
some conflict eh?

**Comment by pveyes** [comment]
Oh no

**Comment by resir014** [comment]
We're getting more failed deployments. @tibudiyanto can you pls check what it is?

**Comment by tibudiyanto** [comment]
image


It is API side.

**Comment by tibudiyanto** [comment]
> 
> 
> We're getting more failed deployments. @tibudiyanto can you pls check what it is?

https://github.com/tibudiyanto/jakarta-vax-availability/pull/51

this seems to address it

**Comment by resir014** [comment]
yep, that fixed it! deploys should work again now.

**Pull request merged**